app/views/students/shared/_student_list.html.erb

```diff
@@ -0,0 +1,77 @@
+    <% @students.each do |student| %>

```

Comment:
You might want to use the render partial for collection method here instead of doing an `.each` on `@students`:

http://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections